.github/workflows/build-and-release.yaml

```
@@ -110,6 +110,7 @@ jobs:
           echo "CONFIG_MEMFAULT_NCS_FW_VERSION=\"${{ env.APP_VERSION }}\"" >> firmware.conf
           echo "CONFIG_MEMFAULT_NCS_FW_TYPE=\"thingy_world\"" >> firmware.conf
           echo "CONFIG_MEMFAULT_NCS_PROJECT_KEY=\"${{ env.MEMFAULT_PROJECT_KEY }}\"" >> firmware.conf
+          echo MEMFAULT_OVERLAY=/workdir/project/overlay-memfault.conf >> $GITHUB_ENV
```

File does not exist in source. I guess you meant to use https://github.com/nrfconnect/sdk-nrf/blob/7de4f07a1fb5106f996c9bad1a3dd92b807da537/applications/asset_tracker_v2/overlay-memfault.conf

```suggestion
          echo MEMFAULT_OVERLAY=overlay-memfault.conf >> $GITHUB_ENV
```